### PR TITLE
onclick instead of ontap for refresh and export

### PR DESCRIPTION
**Pull request by Georgehe4**

### CODE REVIEW

**Comment by Georgehe4** [comment]
@devoncarew

**Comment by devoncarew** [comment]
Awesome! Can you also convert over the over onTap calls to onClick in mobile.dart and embed.dart? I think we could do with a common utility method here, as I think we're also going to want to indicate that the event is handled by our button listener:

```
e.stopPropagation();
e.stopImmediatePropagation();
```

**Comment by kasperpeulen** [comment]
Why do you change ontap to onclick ? I read that ontap works the same as onclick except that it should also be more touch friendly.

**Comment by Georgehe4** [comment]
@kasperpeulen 
We're seeing some inconsistencies on when the onTap method is being executed - this has caused some issues with focusing and registering tap events twice.

File: lib/embed/embed.dart [hunk 1]
```diff
@@ -116,7 +116,15 @@ class PlaygroundMobile {
 
     _showGist(gistId, run: page == 'run');
   }
-
+  
+  void clickAction(a, Function f) {
```
Comment (devoncarew) [inline lib/embed/embed.dart]: Can you add a type annotation to `a`? A `WebElement` or a `PolymerElement`.

File: lib/mobile/mobile.dart [hunk 2]
```diff
@@ -125,6 +125,14 @@ class PlaygroundMobile {
     //_router.go('gist', {'gist': currentGistId()});
   }
 
+  void clickAction(a, Function f) {
```
Comment (devoncarew) [inline lib/mobile/mobile.dart]: same

Comment (devoncarew) [inline lib/mobile/mobile.dart]: Or, move this util code to `base.dart` or `polymer.dart`?

File: web/embed_style.html [hunk 3]
```diff
@@ -20,6 +20,7 @@
     --paper-tabs-selection-bar-color: var(--light-primary-color);
     --paper-tab-ink: var(--light-primary-color);
     --paper-fab-background: var(--accent-color);
+    --paper-fab-keyboard-focus-background: var(--accent-color);
```
Comment (devoncarew) [inline web/embed_style.html]: :)

**Comment by devoncarew** [comment]
lgtm!

@kasperpeulen, if you have ideas about when we're getting double notifications for `onTap` events, we're all ears :) I know you've been pretty deep into polymer recently.

**Comment by kasperpeulen** [comment]
@devoncarew I can remember I had a issue with onTap when I was building the polymer mobile for 1.0 some time ago, for what I remember that was solved with 
e.stopImmediatePropagation();

**Comment by kasperpeulen** [comment]
see: https://github.com/kasperpeulen/dart-pad/commit/22af26424c3b1085d6cb23025f55495718e1904a#diff-be39108aa992e594d7f84cedea214eb7R192

**Comment by Georgehe4** [comment]
@kasperpeulen 
I will try out the onClick methods for now and look to switch back to onTap if it works better in the future in order to fix some issues for a live push.
Even with stopPropagation/stopImmediatePropagation, we ran into an issue where we were losing focus on a paper dialog on mobile.

**Comment by Georgehe4** [comment]
Merging.

**Comment by kasperpeulen** [comment]
okay sounds reasonable, btw, I can recommend this slack channel:
https://polymer.slack.com/messages

that is where I get the tip to use e.stopPropagation (from a polymer team member)

**Pull request merged**